neural_seq_qa/network.py

```diff
@@ -0,0 +1,314 @@
+import math
+import paddle.v2 as paddle
+
+import reader
+
+__all__ = ["training_net", "inference_net", "feeding"]
+
+feeding = {
+    reader.Q_IDS_STR: reader.Q_IDS,
+    reader.E_IDS_STR: reader.E_IDS,
+    reader.QE_COMM_STR: reader.QE_COMM,
+    reader.EE_COMM_STR: reader.EE_COMM,
+    reader.LABELS_STR: reader.LABELS
+}
+
+
+def get_embedding(input, word_vec_dim, wordvecs):
+    """
+    Defined word embedding
+    
+    :param input: layer input
+    :type input: LayerOutput
+    :param word_vec_dim: dimension of the word embeddings
+    :type word_vec_dim: int
+    :param wordvecs: word embedding matrix
+    :type wordvecs: numpy array
+    :return: embedding
+    :rtype: LayerOutput
+    """
+    return paddle.layer.embedding(
+        input=input,
+        size=word_vec_dim,
+        param_attr=paddle.attr.ParamAttr(
+            name="wordvecs", is_static=True, initializer=lambda _: wordvecs))
+
+
+def encoding_question(question, q_lstm_dim, latent_chain_dim, word_vec_dim,
+                      drop_rate, wordvecs, default_init_std, default_l2_rate):
+    """
+    Define network for encoding question
+
+    :param question: question token ids
+    :type question: LayerOutput
+    :param q_lstm_dim: dimension of the question LSTM
+    :type q_lstm_dim: int
+    :param latent_chain_dim: dimension of the attention layer
+    :type latent_chain_dim: int
+    :param word_vec_dim: dimension of the word embeddings
+    :type word_vec_dim: int
+    :param drop_rate: dropout rate
+    :type drop_rate: float
+    :param wordvecs: word embedding matrix
+    :type wordvecs: numpy array
+    :param default_init_std: default initial std
```

Comment:
Please use the full name of std in all files.

Reply:
done